server/services/votes/whovoted.service.js

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
I think it makes more sense to add this as a field to a company, using the `alterItems` or `fastJoin` hook to tabulate the counts within the companies service. It reduces a round trip and the number of APIs we have to think about.